schemas/questionnaire_v1.json

```
@@ -62,21 +62,37 @@
             "type": "string",
             "pattern": "^[a-zA-Z0-9-_]+$"
           },
-          "validator": {
+          "type": {
             "type": "string",
             "enum": [
               "boolean",
               "date",
               "string",
-              "optional_string",
-              "uuid"
+              "uuid",
+              "url"
             ]
+          },
+          "min_length": {
+            "type": "number",
```

Use integer instead?
```suggestion
            "type": "integer",
```